sources/payment-hub/target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst

```diff
@@ -0,0 +1,153 @@
+hu/dpc/rt/psp/config/BindingProperties.class
+hu/dpc/rt/psp/config/SwitchSettings.class
+hu/dpc/rt/psp/type/TransactionState$1.class
```

Comment:
Again, looks like a build script generated file which shouldn't be added to source control.